### DISCUSSION TITLE
[Feature] Favourite/Categrorise/Sort people

### DISCUSSION THREAD

**Discussion by RedCommander735** [immich-app/immich#10388] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It is sometimes really hard to find a person in the people's tab if you have too many.
I think it would be really useful, if you could favourite people, so they show up above all the others (while still sorted by asset count) so you have your favourites and the immediately after all the other people.
Another way would be to be able to group people (so e.g. you have one group 'family', one 'friends', etc.) and have them show up kinda like a day in the timeline with all people in that category below.

It would also be really cool if you could also sort people by alphabet instead of just by asset count.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#6079
